package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "shop-analytics",
-  "version": "1.7.0",
+  "version": "1.7.2",
```

Should be `1.7.0`? A fixup is not raising the patch version as this PR will be sqash merged and therefore results in a single commit
```suggestion
  "version": "1.7.0",
```